src/Workleap.DomainEventPropagation.Subscription.PullDelivery/CloudEventHandler.cs

```diff
@@ -11,71 +11,53 @@ internal sealed class CloudEventHandler : BaseEventHandler, ICloudEventHandler
     public CloudEventHandler(
         IServiceProvider serviceProvider,
         IDomainEventTypeRegistry domainEventTypeRegistry,
-        IEnumerable<IDomainEventBehavior> domainEventBehaviors,
+        IEnumerable<ISubscriptionDomainEventBehavior> domainEventBehaviors,
         ILogger<ICloudEventHandler> logger)
         : base(serviceProvider, domainEventTypeRegistry)
     {
         this._logger = logger;
         this._pipeline = domainEventBehaviors.Reverse().Aggregate((DomainEventHandlerDelegate)this.HandleDomainEventAsync, BuildPipeline);
     }
 
-    public async Task<EventProcessingStatus> HandleCloudEventAsync(CloudEvent cloudEvent, CancellationToken cancellationToken)
+    public async Task HandleCloudEventAsync(CloudEvent cloudEvent, CancellationToken cancellationToken)
     {
-        try
-        {
-            if (!TryWrapCloudEvent(cloudEvent, out var domainEventWrapper))
-            {
-                this._logger.IllFormedCloudEvent(cloudEvent.Id);
-                return EventProcessingStatus.Rejected;
-            }
-
-            if (this.GetDomainEventType(domainEventWrapper!.DomainEventName) == null)
-            {
-                this._logger.EventDomainTypeNotRegistered(domainEventWrapper.DomainEventName, cloudEvent.Subject ?? "Unknown");
-                return EventProcessingStatus.Rejected;
-            }
-
-            return await this._pipeline(domainEventWrapper, cancellationToken).ConfigureAwait(false);
-        }
-        catch (Exception e)
+        var domainEventWrapper = WrapCloudEvent(cloudEvent);
+        if (this.GetDomainEventType(domainEventWrapper!.DomainEventName) == null)
```

Comment:
Selon la signature de `WrapCloudEvent` tu peux pas retourner un null
```suggestion
        if (this.GetDomainEventType(domainEventWrapper.DomainEventName) == null)
```